.asf.yaml

```diff
@@ -18,6 +18,19 @@
 github:
   description: "Apache Arrow is the universal columnar format and multi-language toolbox for fast data interchange and in-memory analytics"
   homepage: https://arrow.apache.org/
+  labels:
+    - arrow
+    - parquet
+
+  features:
+    issues: true
+    projects: true
+    discussions: true
+
+  protected_branches:
+    # protect main against force push
+    main: {}
```

Comment:
do we want to manually unprotect the branch when we have to push the bump versions?
There is currently a requirement on the release process to push to main for the bump versions:
https://github.com/apache/arrow/blob/main/dev/release/post-12-bump-versions.sh#L78

Reply:
This only prevents force push, you can still push directly to main, I know that is the reason we previously didn't use branch protection.